jobs/merge-branches.groovy

```diff
@@ -96,9 +96,10 @@ onMaster('1h') {
                    when: ['FAILURE', 'UNSTABLE']]]) {
       try {
          checkArgs();
-         tag_name = ("buildmaster-${params.COMMIT_ID}-" +
+         tagName = ("buildmaster-${params.COMMIT_ID}-" +
```

Comment:
groovy convention would keep this as `tag_name`...

Reply:
Let me try saying it again hopefully with more clarity :)

I am looking at the convention in the file so I think we want to be consistent in the file so that we dont have some variables that are snake and some camel all in the same file.  I thought I saw a mismatch...

But the general repo convention matches more with the change you made.  So if you want to change the entire file, that seems great to me!  Im just looking for consistency starting from file.  And hopefully the file matches the rest of the code in the repo.  If not, we can have a separate PR for updating the style.